.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ parameters:
     default: ""
   REPOSITORY:
     type: string
-    default: "https://github.com/thales-markets/thales-sport-markets"
+    default: "https://github.com/jlmonroy13/vite-project"
```

Comment:
now that we have the landing site public maybe we can use that? 🙂

Reply:
oh yep, I forgot that, sorry